app/src/main/java/org/mozilla/focus/fragment/HomeFragment.kt

```diff
@@ -117,6 +124,13 @@ private class HomeTileAdapter(val onUrlEnteredListener: OnUrlEnteredListener, ho
             onUrlEnteredListener.onNonTextInputUrlEntered(item.url)
             TelemetryWrapper.homeTileClickEvent()
         }
+        itemView.setOnLongClickListener {
+            deleteTile(item.id)
```

Comment:
Nit: be more specific in this method name - probably `addBundledTileToBlacklist` rather than `deleteTile`. It's not clear what "delete" means, does it mean that you're removing the tile from UI too?

Write maintainable code! That means naming methods to what they actually do - in the future, someone might see `deleteTile` and add some code there that belongs somewhere else, but if you name it `addBundledTileToBlacklist` no one will be confused.  ("Bundled" is optional, up to you)